features/networking/sdn.feature

```diff
@@ -340,4 +340,34 @@ Feature: SDN related networking scenarios
       | curl | --connect-timeout | 5 | <%= cb.pod4_ip %>:8080 |
     Then the step should succeed
     And the output should contain "Hello OpenShift"
-
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-25787
+  @admin
+  Scenario: Don't write CNI configuration file until ovn-controller has done at least one iteration
+    Given the env is using "OVNKubernetes" networkType
+    And I select a random node's host
+    #Checking controller iteration 1
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ovn-appctl -t ovn-controller connection-status |
+    Then the step should succeed
+    And the output should contain "connected"
+    #Checking controller iteration 2
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ls -l /var/run/ovn/ |
```

Comment:
do we always need the `bash -c`?

Reply:
Normally if we have a bash expression with redirects, pipes, etc. then we use bash as command and pass expression to bash with the -c param to run that whole command as one in subshell. We have seen issues in past if not use its specially when executing on pods. Here it may not be needed as i don't see any pipes,redirection,quotes etc. But guess most of us  are just accustomed to use bash -c now :)